c/go/Makefile

```diff
@@ -1,16 +1,17 @@
 all: build
 
-build: libvirtblocks_c_go.la capi.lo
+build: libvirtblocks_c_go.la
 
 gobuild:
 	go build -buildmode c-archive -o libvirtblocks_c_go.a capi.go
 
-capi.lo: capi.c gobuild
-	libtool --mode=compile --tag=CC gcc -c capi.c
+capi.o: capi.c gobuild
+	gcc -c capi.c -o capi.o
 
-libvirtblocks_c_go.la: ../libtool.la.in gobuild
+libvirtblocks_c_go.la: ../libtool.la.in gobuild capi.o
 	mkdir -p .libs/
-	ln -sf ../$(@:.la=.a) .libs/$(@:.la=.a)
+	cp $(@:.la=.a) .libs/$(@:.la=.a)
+	ar r .libs/$(@:.la=.a) capi.o
```

Comment:
I assume you know what you're doing, but how come we're only adding `capi.o` to the `libvirtblocks_c_go.a` that lives in `.libs/` and not to the one in the current directory?

Reply:
It's in the commit message, but basically I saw the output of the go build as an intermediate step and the archive in `.libs/` as the output of the next step. I am also copying the resulting archive produced by go instead of symlinking it just to make sure everything works in case go actually does not regenerate the output file from scratch. I wanted to be on the safe side, because I am not sure how go was doing that. If you want I can keep symlinking it and just add the capi.o to the same file.

Reply:
Yeah, let's go with the symlink.